### DISCUSSION TITLE
Scale or position issues

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3770] [Q&A] [answered]
Hi all,

I have an AR positioning issue ( or it may be scale). Can you help?

 - _Request feature add in the model viewer editor a View in AR button_

I have the 3d model out of C4D with a scale close to the reality of a window! With the root set to (0,0,0) in the middle bottom of the model - as shown

![Root Window](https://user-images.githubusercontent.com/17274973/188147123-60b5fbc0-ef5c-44e3-ba47-0a412ae9a684.png)

To double-check the  size I passed through Sketchfab and it looks good ( position and scale)

![sckfab](https://user-images.githubusercontent.com/17274973/188148535-f4f55360-f136-4f05-a799-62f9d9087049.png)

When I add it to my project with "webxr scene-view quick-look" it shows the model for a quick sec. With " scene-view" only the model scale is huge.

![Screen_Recording_20220902-152714_Chrome (1)](https://user-images.githubusercontent.com/17274973/188150276-688378c1-6d1e-45af-a2a5-a06279a69527.gif)

Code is Here: https://glitch.com/edit/#!/lems-window-test
Live site: https://lems-window-test.glitch.me

Device: Samsung S20/ android 12 / Browser: Chrome 104.0.5112.97

**Comment by elalish** [comment, marked as answer]
It definitely appears to be too big (wrong units). glTF is always in meters. Not sure why Sketchfab's tool isn't catching it. You can load it in our [editor](https://modelviewer.dev/editor/) and click "model details" (or warning/error, depending on the validation) to see the bounding box size. I also just fixed a related bug for quantized models, so if it's working in the editor, we'll have a release out with the fix soon.

**Comment by rossrossp** [comment]
I have this same issue with .glb/.gltf files exported from Blender.
Even if I export a standard 1m cube from Blender it's huge in AR when I deploy it to mobile via the Model Editor.

Here it is shrunk to 50% in AR. The bottom of the guitar is 99cm from the ground for reference:
![IMG_4731](https://user-images.githubusercontent.com/6325131/188328110-74134638-04ba-46a4-8a8a-b61fd79ec1c2.JPG)

The editor's validation report for the default Blender cube is as follows:
```
Format: glTF 2.0
Generator: Khronos glTF Blender I/O v1.6.16
Stats:
1 draw calls
0 animations
0 joints
1 materials
24 vertices
12 triangles
2.00 m x-width
2.00 m y-height
2.00 m z-length
```
Which, unless I misunderstand, does imply that the model is twice the size it should be.
Hopefully this helps with diagnosing/fixing the issue, or someone can point me to something I'm doing wrong.

**Reply by elalish** [reply]
It does appear to be a 2m cube; are you sure it's really meant to be a 1m cube and not a cube that goes from -1 to 1? Also, have you moved around in AR enough to actually place the object on the floor? It might just be closer to you than expected.
